Adyen/UI/UI Style/FormComponentStyle.swift

```diff
@@ -8,10 +8,15 @@ import Foundation
 import UIKit
 
 /// Contains the styling customization options for any form-based component.
-public struct FormComponentStyle: ViewStyle {
+public struct FormComponentStyle: FormValueItemStyle {
     
     /// :nodoc:
     public var backgroundColor = UIColor.Adyen.componentBackground
+
+    /// The section header style.
+    public var sectionHeader = TextStyle(font: .preferredFont(forTextStyle: .headline),
```

Comment:
can this be a constant? maybe have one designated initializer where fields are set to passed (or default) values. and a few convenience initializers, that call designated with default parameters

Reply:
It should be, sure.
But we better do it in other PR 😝